src/Patterns/Path.php

```diff
@@ -322,4 +322,92 @@ private function conditionToString(): string
 
         return sprintf("[%s]", $conditionInner);
     }
+
+    /**
+     * @return MapType|null
+     */
+    public function getProperties(): ?MapType
+    {
+        return $this->properties;
+    }
+
+    /**
+     * Returns the variable of the path.
+     *
+     * @return Variable
+     */
+    public function getVariable(): ?Variable
+    {
+        return $this->variable;
+    }
+
+    /**
+     * Returns the left structure of the relationship.
+     *
+     * @return StructuralType
+     */
+    public function getA(): StructuralType
+    {
+        return $this->a;
+    }
+
+    /**
+     * Returns the right structure of the relationship.
+     *
+     * @return StructuralType
+     */
+    public function getB(): StructuralType
```

Comment:
I think it would be better to rename `$this->b` to `$this->right` and rename this function to `getRight`.

Reply:
Fixed in 4ff823a